.github/workflows/regression-test-docker-image.yml

```diff
@@ -74,16 +74,16 @@ jobs:
     - name: Check if webperf_core can run Test 1
       if: always()
       run: |
-        testresult=$(docker run ${{ env.TEST_TAG }} python default.py -t 1 -r -i sites.json)
+        testresult=$(docker run --shm-size=4g -e MAX_OLD_SPACE_SIZE=3000 ${{ env.TEST_TAG }} python default.py -t 1 -r -i sites.json)
```

Comment:
@krompaco cool, maybe we should set it using ENV in docker file instead?
What is the lowest value that works?

Reply:
@krompaco IF arguments you use here is required (I would prefer a way it was not required), this PR should also include modification of docker documentation to include this.

Reply:
They shouldn't be required on "larger" host machines but seems to be needed for GH Actions where heavier work is done. Values are dependent on host machine resources, so not a good candidate to "burn" into the image.